flask_swagger.py

```diff
@@ -17,10 +17,42 @@ def _sanitize(comment):
     return comment.replace('\n', '<br/>') if comment else comment
 
 
-def _parse_docstring(obj, process_doc):
+def _find_from_file(full_doc, from_file_keyword):
+    """
+    Finds a line in <full_doc> like
+
+        <from_file_keyword> <colon> <path>
+
+    and return path
+    """
+    path = None
+
+    for line in full_doc.splitlines():
+        if from_file_keyword in line:
+            parts = line.strip().split(':')
+            if len(parts) == 2 and parts[0].strip() == from_file_keyword:
+                path = parts[1].strip()
+                break
+
+    return path
+
+
+def _doc_from_file(path):
+    doc = None
+    with open(path) as f:
+        doc = f.read()
+    return doc
+
+
+def _parse_docstring(obj, process_doc, from_file_keyword):
     first_line, other_lines, swag = None, None, None
     full_doc = inspect.getdoc(obj)
     if full_doc:
+        from_file = _find_from_file(full_doc, from_file_keyword)
```

Comment:
Check for `from_file_keyword is None` in which case just skip the _find_from_file process